backend/api/src/main/kotlin/software/shonk/adapters/incoming/ShorkInterpreterController.kt

```diff
@@ -18,6 +18,20 @@ fun Route.configureShorkInterpreterControllerV1() {
     val logger = LoggerFactory.getLogger("ShorkInterpreterControllerV1")
     val shorkUseCase by inject<ShorkUseCase>()
 
+    /**
+     * Returns the player program code from a lobby which was specified in the path parameters. Path
+     * parameter - {lobbyId}: The lobby id from which the player code has to be gotten. Path
+     * parameter - {player}: The player path variable has to be one of [A,B].
+     *
+     * Response 200: The body contains the player code from the player, which was specified in the
+     * path. response: { "code": String, }
+     *
+     * Response 400: An incorrect player has been specified in the path/request. response: {
+     * "message": String, }
+     *
+     * Response 404: No code has been submitted for the specified player yet or the lobby does not
+     * exist.
```

Comment:
We only specified 404 in case the lobby does not exists. If no player code has been submitted, I think it should just be 200 with empty string for code value. So my guess would be to just reduce this to "...if the lobby does not exist"